audio/pcaudiolib/Portfile

```diff
@@ -0,0 +1,36 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           github 1.0
+
+name                pcaudiolib
+github.setup        espeak-ng pcaudiolib 1.1
+version             1.1
+revision            1
+categories          audio
+platforms           darwin
+maintainers         nomaintainer
+license             GPL-3+
+
+description         Portable C Audio Library
+
+long_description    Portable C Audio Library
```

Comment:
The long description should be full sentences, such as `${name} is a portable C audio library.` That's a fairly short long description though. If more information is available, you could add a sentence or two more.

Reply:
Updated from the README, but it is still somewhat short.